pkgs/by-name/cn/cnsprcy/package.nix

```
@@ -1,33 +1,51 @@
 {
-  lib,
+  lib, # Dependencies/arguments
   fetchFromSourcehut,
   rustPlatform,
   sqlite,
 }:
 
-rustPlatform.buildRustPackage {
+rustPlatform.buildRustPackage rec {
+  # Main package definition
```

We can use finalAttrs here since `buildRustPackage` supports it, which is better for overriding packages than `rec`, which has its own [pitfalls](https://nix.dev/guides/best-practices#recursive-attribute-set-rec).

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```